src/surf_react_adsorb.h

```diff
@@ -192,6 +192,8 @@ class SurfReactAdsorb : public SurfReact {
     double v[3];            // particle velocity
     double erot;            // rotational energy
     double evib;            // vibrational energy
+    double eelec;
```

Comment:
New variables don't seem to be used?

Reply:
Another leftover from before the custom data transition. A question I have though, does the call to `particle->add_particle` in line 1230 ever trigger the `update_custom`? It is easy to default to 0 energy in `update_custom` but we may have cruft left over if particles are being deleted/created quickly (this may be answered by our previous emails about vibrational energy).